faq/dynamic-faq-test.js

```diff
@@ -136,13 +144,10 @@ function sortFaqContentByOrder( a, b ) {
 
 const loadFaq = async () => {
 	let url ="https://covid-taichung.github.io/cfiw/faqDataJSON.json";

```

Comment:
this url needs updating to:
`let url ="https://covid-taichung.github.io/cfiw/faq/faqDataJSON.json";`